Lection 4

dezonnov1 opened the pull request.

dezonnov1 committed:
test lection_1

dezonnov1 committed:
test lection 1 branch

dezonnov1 committed:
test lection 1 branch

dezonnov1 committed:
надо доделать gun.mod, smg и взаимодействие между Player и Weapon

dezonnov1 committed:
* иррархия weapon -> gun -> smg получается 2 потомка
* интерфейса и каждый из них реализован(shooting в gun, а modificate в weapon)
* абстрактный класс entity c абстрактной функцией printInfo()
* КАЖДЫЙ класс содержит >2 полей
* в каждом классе есть параметризированный конструктор
* реализация инкапсуляции в entity, класс coords, где сеттеры posX posY являются private
* статическое поле и метод в entity, counter и displayCounter()
* методы в каждом классе
* методы реализовывают какую-то сложную логику

dezonnov1 committed:
Испрвление замечаний
Добавление пустых строчек в конце кода классов.
Пополнение gitignore
javadoc

dezonnov1 committed:
Испрвление замечаний
Добавление пустых строчек в конце кода классов.
Пополнение gitignore
javadoc

dezonnov1 committed:
Испрвление замечаний
Добавление пустых строчек в конце кода классов.
ИСПРАВЛЕНИЕ gitignore
javadoc

dezonnov1 committed:
Delete .idea directory

dezonnov1 committed:
Update .gitignore

dezonnov1 committed:
1 задание есть!

dezonnov1 committed:
1 задание есть!

dezonnov1 committed:
2 задание есть!

dezonnov1 committed:
Выполнено

dezonnov1 committed:
поправка javadoc

dezonnov1 committed:
4 задание есть!

DS-170 reviewed (changes requested):
Работа не соответствует некоторым условиям задачи:
- Написать метод, который удаляет студентов со средним баллом <3.

Аня не удаляется из списка, хоть и не переходит на следующий курс

dezonnov1 committed:
Исправление методов в DeanOffice.java
transferToOlderCourse()
delBadStudents()
Теперь методы выполняют условия задания

dezonnov1 commented:
> Работа не соответствует некоторым условиям задачи:
> 
>     * Написать метод, который удаляет студентов со средним баллом <3.
> 
> 
> Аня не удаляется из списка, хоть и не переходит на следующий курс

Исправил.
Забыл, что Java всегда передает в метод значения.

dezonnov1 committed:
Удаление ненужных классов для этого задания
